docs/variants/clevo_nv41/flashing_external.md

```diff
@@ -17,6 +17,31 @@ Requirements:
 Ensure that the EC is running updated firmware. Follow the steps outlined in
 [EC firmware update](../ec_update)
 
+## Build flashrom
+
+Currently, the latest flashrom release lacks support for Tiger Lake-U flash
```

Comment:
@mkopec We do not neet chipset support for external flashing, roight?

Reply:
We need it for flashrom to recognize the tiger lake flash descriptor, even when flashing externally. Otherwise we can't flash the BIOS region alone

Reply:
ok